build_variables.bzl

```diff
@@ -1401,11 +1401,14 @@ aten_native_source_non_codegen_list = [
     "aten/src/ATen/native/mkl/SparseBlasImpl.cpp",
     "aten/src/ATen/native/mkl/SparseCsrLinearAlgebra.cpp",
     "aten/src/ATen/native/mkl/SpectralOps.cpp",
+    "aten/src/ATen/native/nested/NestedTensorAliases.cpp",
```

Comment:
Should be the only changes needed

Reply:
We'll probably also need to update TARGETS

Reply:
You were right. Internal build passed with these changes only.